frontend/src/layouts/content-layout.vue

```diff
@@ -121,6 +122,10 @@ export default defineComponent({
         isDesktopLayout.value
     )
 
+    const bannerHeight = computed(() => {
+      return "--banner-height: " + uiStore.shownBannersCount * 4 + "rem;"
+    })
```

Comment:
Styles can take an object, it's more readable.

```suggestion
    const appStyles = computed(() => {{
      "--banner-height": `${uiStore.shownBannersCount * 4}rem`
    }})
```